### PR TITLE
Working

### PR DESCRIPTION
**Pull request by PalmTreeVI**
Restructure pages for better flow, remove unneccessary pages

### CODE REVIEW

File: 02.wallets/01.pivx-core-wallet/02.core-wallet-faq/article.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+4. Go to Settings --> Debug --> Wallet Repair --> Delete local blockchain.
+5. Wait for resync to complete.
+
+From the command line, stop the wallet then run the command `pivxd -daemon -resync`
+[/faq]
+[faq question="Does PIVX provide official blockchain snapshots? How do I create my own blockchain snapshot?"]
+No, there is no official blockchain snapshot.
+
+You 
+can however create your own if you don't want to have to download the full blockchain in case you need to reinstall the core wallet. Here are the steps:
```
Comment (Liquid369) [inline 02.wallets/01.pivx-core-wallet/02.core-wallet-faq/article.md]: I guess a newline happened here accidentally?

**Review by Liquid369 — CHANGES_REQUESTED** [review]
Outside of that newline, everything looks great, lets adjust that and then I will utACK/approve

**Review by JSKitty — DISMISSED** [review]
**utACK**, +1 on Liquid's find, I could not find any other obvious flaw after reading a fair few of the pages, this is an **AWESOME** rework, great job! :tada: :purple_heart:

File: 02.wallets/01.pivx-core-wallet/02.core-wallet-faq/article.md (hunk 1 above)
Comment (PalmTreeVI) [reply 02.wallets/01.pivx-core-wallet/02.core-wallet-faq/article.md]: I forgot to finish editting the page, was supposed to change snapshots faq, newline fixed and extra text added.

File: 02.wallets/02.my-pivx-wallet/article.md [hunk 2]
```diff
@@ -3,13 +3,14 @@ title: 'My PIVX Wallet'
...
 MPW is a light wallet (the entire blockchain does not need to be downloaded).  While it is feature rich, it does not currently have all the functionality of the PIVX Core wallet.
 
-You can find MPW at https://mypivxwallet.org.
-
-Follow the progress at  https://github.com/PIVX-Labs/MyPIVXWallet.
+You can find the official MPW website at https://mypivxwallet.org.
 
+Follow MPW progress at:
+* https://github.com/PIVX-Labs/MyPIVXWallet
+* https://medium.com/@pivx-labs
```
Comment (JSKitty) [inline 02.wallets/02.my-pivx-wallet/article.md]: I'd also suggest having our Discord here: https://discord.gg/NkGqS9UJC8

You can hit "Commit Suggestion" for ease.
```suggestion
* https://medium.com/@pivx-labs
* https://discord.gg/NkGqS9UJC8
```

**Review by JSKitty — DISMISSED** [review]
**utACK** c517e9a5eddce4e9c04e9d0745ea75c991d364a9!

Only a minor/optional suggestion to add the Labs Discord link along the "GitHub" and "Medium" ones.

**Review by JSKitty — DISMISSED** [review]

**Review by Liquid369 — APPROVED** [review]
tACK 70122e3704c1b3ac6f86d8e54a0d04d0ef0aded2

**Review by JSKitty — APPROVED** [review]
**utACK** 70122e3704c1b3ac6f86d8e54a0d04d0ef0aded2 :tada:

**Pull request merged**